ambuda/seed/dictionaries/shabdartha_kaustubha.py

```diff
@@ -43,12 +44,14 @@ def sak_generator(dict_blob: str):
             buf = []
 
 
-def run():
+@click.command()
+@click.option("--use-cache/--no-use-cache", default=False)
```

Comment:
Just checking: have you thought about the pros-and-cons of having the default be to not use the cache? (Can see good points either way, so as long as this has been considered I'd be fine with either option…)

Reply:
Yes -- the main advantage of the cache is for development, so that we can run a script N times without waiting for N network fetches. For a prod script I don't think it's that useful since the script will be run just once, and in fact we should consider avoiding it in prod so that the script has a more predictable data flow.